effekt/shared/src/main/scala/effekt/RecursiveDescent.scala

```diff
@@ -1395,25 +1407,27 @@ class RecursiveDescent(positions: Positions, tokens: Seq[Token], source: Source)
 
   inline def braces[T](p: => T): T =
     consume(`{`)
-    val res = p
+    val res =  p
```

Comment:
This might be accidental
```suggestion
    val res = p
```